core/relay/config/Corda_Relay.toml

```diff
@@ -5,6 +5,10 @@ hostname="localhost"
 db_path="db/Corda_Relay/requests"
 # This will be replaced by the task queue.
 remote_db_path="db/Corda_Relay/remote_request"
+# max retries opening sled db if it is locked
+db_open_max_retries=500
+# retry back off time in ms if sled db is locked
+db_open_retry_backoff_time=10
```

Comment:
Aren't we using 20ms and 250 retries as the default?

Reply:
For relay, since db is opened for just database operations, and not whole procedure call, and also thought rust would be faster, hence kept 500 retries with 10ms for relay.